src/frontend/app/features/applications/application/application-tabs-base/application-tabs-base.component.ts

```diff
@@ -57,8 +58,8 @@ export class ApplicationTabsBaseComponent implements OnInit, OnDestroy {
   constructor(
     private route: ActivatedRoute,
     private router: Router,
-    private applicationService: ApplicationService,
-    private entityService: EntityService<APIResource>,
+    public applicationService: ApplicationService,
+    @Inject(ENTITY_SERVICE) private entityService: EntityService<APIResource>,
```

Comment:
Q.. Here we've added inject... however in the same PR we've removed the service's injectable. How come?